examples/CV7/CV7_example.cpp

```diff
@@ -168,64 +188,95 @@ int main(int argc, const char* argv[])
     event_params.threshold.high_thres = -0.7853981;
     event_params.threshold.low_thres  = 0.7853981;
 
-    if(commands_3dm::writeEventTrigger(*device, FILTER_ROLL_EVENT_ACTION_ID, commands_3dm::EventTrigger::Type::THRESHOLD, event_params) != CmdResult::ACK_OK)
```

Comment:
This line wrap is very confusing. Why is CmdResult::ACK_OK on it's own indentation level?